constructorio-client/src/main/java/io/constructor/client/models/BaseResultsResponse.java

```diff
@@ -0,0 +1,78 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+
+/** Constructor.io Base Result ... uses Gson/Reflection to load data in */
+public class BaseResultsResponse {
+
+    @SerializedName("facets")
+    private List<FilterFacet> facets;
+
+    @SerializedName("groups")
+    private List<FilterGroup> groups;
+
+    @SerializedName("results")
+    private List<Result> results;
+
+    @SerializedName("total_num_results")
+    private Integer totalNumberOfResults;
+
+    @SerializedName("sort_options")
+    private List<FilterSortOption> sortOptions;
+
```

Comment:
Forgot one thing, do we want to move `ResultSources` here?

Reply:
Added